knowledge-base/how-to-modify-graph-axis-labels.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Modifing Graph Axis labels
+description: Customize the Axis label
+type: how-to
+page_title: How to Modify Graph Axis Labels
+slug: how-to-modify-graph-axis-labels
+position: 
+tags: graph
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
```

Comment:
Add more details in the environment table. The Graph is a report element for example